utils/utils.go

```diff
@@ -49,3 +51,40 @@ type SetupWebhookWithManagerConfig struct {
 	HandlerEnabledFlag     bool
 	DefaultDurationFlag    time.Duration
 }
+
+// GetTargetedContainersInfo gets the IDs of the targeted containers or all container IDs found in a Pod
+func GetTargetedContainersInfo(pod *corev1.Pod, targets []string) (map[string]string, error) {
```

Comment:
Moved the function from controllers/helper to utils in order to use it in the injection logic when reinjection